dist/css_tagger.min.js

```diff
@@ -1,3 +1 @@
-// Minified on http://closure-compiler.appspot.com/ using Advanced settings
```

Comment:
Not at all, although the ideal thing would be for the `gulp dist` task to create it automatically. Currently, `gulp clean` deletes the file. This should be addressed in a separate PR IMO.